data-migration/migrate-with-binlog-event-filter.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Filter Binlog Events
+summary: Learn how to filter binlog events when migrating data
+---
+
+# Filter Binlog Events
+
+This document describes how to filter binlog events when you use DM to perform continuous incremental data replication. For the detailed replication instruction, refer to the following documents:
+
+- [Migrate MySQL Data of Less Than 1 TiB to TiDB](/data-migration/migrate-mysql-tidb-less-tb.md)
+- [Migrate MySQL Data of More Than 1 TiB to TiDB](/data-migration/migrate-mysql-tidb-above-tb.md)
+- [Migrate and Merge MySQL Shards of Less Than 1 TiB to TiDB](/data-migration/migrate-shared-mysql-tidb-less-tb.md)
+- [Migrate and Merge MySQL Shards of More Than 1 TiB to TiDB](/data-migration/migrate-shared-mysql-tidb-above-tb.md)
+
+## Configuration
+
+To use binlog event filter, add a `filter` when configuring the task configuration file of DM, as shown below:
```

Comment:
too many "configure" in this sentence, how about "add a `filter` when completing the task configuration file of DM"